src/wh_she_common.c

```diff
@@ -22,6 +22,9 @@
  */
 /* System libraries */
 
+/* Pick up compile-time configuration */
+#include "wolfhsm/wh_settings.h"
+
 #ifdef WOLFHSM_CFG_SHE_EXTENSION
```

Comment:
Why not simply move the ifdef SHE_EXTENSION below the any of the wh_*.h includes below?  I was hoping to avoid having any reference to wh_settings.h in the .c files.